styles.css

```diff
@@ -4,91 +4,92 @@
   box-sizing: border-box;
   margin: 0;
   padding: 0;
+  font-family: 'Poppins', sans-serif;
+  background-repeat: no-repeat;
 }
 
 body {
   background-image: url(./images/image_geometry_1.svg);
-  background-repeat: no-repeat;
 }
 
-/* Navigation section */
-header {
-  display: flex;
-  width: 95vw;
-  height: 50px;
-  justify-content: flex-end;
-  align-items: center;
-}
-
-/* Hero section */
-.hero-container {
-  width: 90vw;
-  height: 90vh;
-  display: flex;
-  flex-flow: column;
-  justify-content: center;
-  gap: 30px;
-  margin: auto;
+.bg-dark {
+  background-color: #1a2236;
 }
 
 .title {
   font-family: 'Playfair Display', serif;
   font-size: 50px;
-  font-style: normal;
   font-weight: 400;
   line-height: 50px;
-  letter-spacing: 0;
   text-align: left;
   color: #1a2236;
 }
 
-.hero-container p {
-  font-family: 'Poppins', sans-serif;
+.paragraph {
   font-size: 14px;
-  font-style: normal;
   font-weight: 400;
   line-height: 21px;
   letter-spacing: -0.01em;
   text-align: left;
 }
 
-button.bg-dark {
-  background-color: #1a2236;
-  font-family: 'Poppins', sans-serif;
+.button {
   font-size: 16px;
-  font-style: normal;
   font-weight: 500;
   line-height: 24px;
-  letter-spacing: 0;
   text-align: center;
-  color: #fff;
+  height: 50px;
+  width: 100%;
 }
 
-.hero-container button.bg-dark {
-  height: 50px;
+.btn-bg-dark {
+  background-color: #1a2236;
+  color: #fff;
   border: 1px solid #fff;
 }
 
-button.bg-dark:hover {
+.btn-bg-dark:hover {
   border: 1px solid #1a2236;
   background-color: #fff;
   color: #1a2236;
 }
 
-button.bg-dark:focus {
+.btn-bg-dark:focus {
   border: 2px solid #1a2236;
 }
 
+/* Navigation section */
+.nav {
+  display: flex;
+  width: 95vw;
+  height: 50px;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+.nav__links {
+  display: none;
+}
+
+/* Hero section */
+.hero-container {
+  width: 90vw;
+  height: 90vh;
+  display: flex;
```

Comment:
In order to ensure the DRY rule principle. It will be nice to only declare the `flex` property once. And then reuse it across the css. Instead of declaring it several times. example
```
:root {
   --flex: flex;
  }

.hero-container {
 other codes...
 display: var(--flex);
 other codes...
}
This is just one way to ensure DRY using variables. There are other ways to achieve the same thing.
```